pom.xml

```diff
@@ -6,7 +6,7 @@
 
 	<groupId>eu.europeana.api.commons</groupId>
 	<artifactId>commons-api-services</artifactId>
-	<version>0.2.9-SNAPSHOT</version>
```

Comment:
please merge the develop branch first to not overwrite the snapshot version in all modules

Reply:
Currently 2.8.0 used from develop

Reply:
I mean 0.2.8